vagrant/_commons.sh

```diff
@@ -90,7 +90,11 @@ function install_k8s {
             echo "download_container: false"
             echo "skip_downloads: false"
             } >> ./inventory/group_vars/all.yml
-            sed -i 's/^etcd_deployment_type: .*$/etcd_deployment_type: host/;s/^kubelet_deployment_type: .*$/kubelet_deployment_type: host/;s/^container_manager: .*$/container_manager: crio/' ./inventory/group_vars/k8s-cluster.yml
+            sed -i 's/^etcd_deployment_type: .*$/etcd_deployment_type: host/' ./inventory/group_vars/k8s-cluster.yml
+            sed -i 's/^kubelet_deployment_type: .*$/kubelet_deployment_type: host/' ./inventory/group_vars/k8s-cluster.yml
+            sed -i 's/^container_manager: .*$/container_manager: crio/' ./inventory/group_vars/k8s-cluster.yml
+            # TODO: https://github.com/kubernetes-sigs/kubespray/issues/4737
```

Comment:
the workaround is to stop either docker or cri-o, depending on which don't need

Reply:
In an All-in-One setup you always need docker for building Envoy and Intel Plugin images. Moreover, [there is some errors during the execution of kubespray if Helm is enabled](https://github.com/kubernetes-sigs/kubespray/issues/4705). IMHO both services have to coexist.

Reply:
@mythi I couldn't find in [podman](https://github.com/containers/libpod/blob/master/cmd/podman/build.go#L274-L302) a way to deal proxies like we do with [docker](https://github.com/electrocucaracha/kubernetes-qat-envoy/blob/master/vagrant/configure-qat-envoy.yml#L32-L42). Maybe we have to use `buildah`, but the version that is installed on CentOS is 1.7.1 which doesn't support [--http-proxy](https://github.com/containers/buildah/blob/master/docs/buildah-bud.md) argument.

Reply:
As suggested, you could do `docker save <tag> | podman ...` so that you don't need proxies.

Reply:
I tested that [change](https://gist.github.com/electrocucaracha/f11f9ed3fd32da6aa8c5c11d17d59152) but it didn't work for me.

Reply:
I built `podman` on Clear and tested it:
```
$ sudo bin/podman load -i ~/intel-qat-plugin.tar
Getting image source signatures
Copying blob 48fed9e79c43 done
Copying blob 01092e5921c5 done
Copying blob 6189abe095d5 done
Copying blob 5c3fa42c4bcf done
Copying blob 87c747af6dc3 done
Copying config 6454a82ee2 done
Writing manifest to image destination
Storing signatures
Loaded image(s): localhost/intel-qat-plugin:devel
$ sudo crictl images
IMAGE                        TAG                 IMAGE ID            SIZE
localhost/intel-qat-plugin   devel               6454a82ee2c6d       36.5MB
```

I noticed `podman` was picky about having ` /etc/containers/policy.json` so I simply `sudo cp /usr/share/defaults/crio/policy.json /etc/containers/`

Reply:
I meant the image is copied and listed in the catalog but the daemonset can't pull it to start it.

Reply:
It might be that localhost registry must be added to known registries in `crio.conf`